src/meta-srv/src/keys.rs

```diff
@@ -174,7 +174,7 @@ impl<'a> TableRouteKey<'a> {
     }
 }
 
-#[derive(Eq, PartialEq, Debug, Hash)]
+#[derive(Eq, PartialEq, Debug, Clone, Hash)]
```

Comment:
better derived `Copy`, so no `clone` is needed

Reply:
The definition of the `Copy` trait is as follows: 
```
pub trait Copy: Clone
```
If implement `Copy`, must implement `Clone`